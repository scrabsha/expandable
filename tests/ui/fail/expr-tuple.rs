#![allow(unused)]

#[expandable::expr]
macro_rules! a {
    () => {
        (,)
    };
}

#[expandable::expr]
macro_rules! b {
    () => {
        (42,, 42)
    };
}

#[expandable::expr]
macro_rules! c {
    () => {
        (a b)
    }
}

fn main() {}
