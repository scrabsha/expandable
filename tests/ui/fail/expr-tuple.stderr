error: Potentially invalid expansion. Expected an expression, an identifier, `::`, `<`, `Self`, `break` or 13 others.
 --> tests/ui/fail/expr-tuple.rs:6:10
  |
6 |         (,)
  |          ^

error: Potentially invalid expansion. Expected an expression, an identifier, `::`, `<`, `Self`, `break` or 13 others.
  --> tests/ui/fail/expr-tuple.rs:13:13
   |
13 |         (42,, 42)
   |             ^

error: Potentially invalid expansion. Expected `!=`, `%`, `&&`, `&`, `*`, `+` or 20 others.
  --> tests/ui/fail/expr-tuple.rs:20:12
   |
20 |         (a b)
   |            ^
